src/locale/localed-util.c

```diff
@@ -568,7 +568,7 @@ int x11_write_data(Context *c) {
 
         fputs("# Written by systemd-localed(8), read by systemd-localed and Xorg. It's\n"
               "# probably wise not to edit this file manually. Use localectl(1) to\n"
```

Comment:
here the comment `It's probably wise not to edit this file manually` makes a ton of sense, because we never read this file, we always overwrite it. vconsole.conf is the primary source of information after all, this xorg drop-in is only generated from it.

Hence, editing vconsole.conf manually is OK, but editing the xorg file is not, you will lose everything once localed rewrites anything.